icons.yml

```diff
@@ -109,9 +109,21 @@ mappings:
   - subclass: community_centre
     iconName: town-hall-11
     color: '#8A9CF6'
-  - subclass: place_of_worship
+  - class: place_of_worship
     iconName: place-of-worship-11
     color: '#8A9CF6'
+  - class: place_of_worship
+    subclass: christian
+    iconName: religious-christian-11
+    color: '#8A9CF6'
+  - class: place_of_worship
+    subclass: muslim
+    iconName: religious-muslim-11
+    color: '#8A9CF6'
+  - class: place_of_worship
+    subclass: jewish
+    iconName: religious-jewish-11
+    color: '#8A9CF6'
```

Comment:
for the retrocompat, it may be better to keep a rule for `subclass: place_of_worship`.
If we do so, we can udpdate the style before or after the tile server without any regression.